### PR TITLE
Add label to parametric pulse schema

### PR DESCRIPTION
**Pull request by nkanazawa1989**
This PR adds `label` to parametric pulse schema.

See discussion in [Qiskit/qiskit-terra/#5365](https://github.com/Qiskit/qiskit-terra/pull/5365).

### CODE REVIEW

File: schemas/examples/qasm_w_pulse_gates.json [hunk 1]
```diff
@@ -293,6 +293,7 @@
               "t0": 0,
               "ch": "d0",
               "pulse_shape": "gaussian",
+              "label": "pulse1",
```
Comment (taalexander) [inline schemas/examples/qasm_w_pulse_gates.json:296]: Please leave one parametric pulse without a label to ensure it passes.

**Review by taalexander — CHANGES_REQUESTED** [review]
Please bump the minor version number for the schema.

**Comment by nkanazawa1989** [comment]
Thanks, I added two pulses with and without label for each example. Also updated minor version (is it minor or revision?)

**Review by taalexander — APPROVED** [review]
Looks good to me, @mtreinish?

**Review by mtreinish — APPROVED** [review]
LGTM

**Pull request merged**